### PR TITLE
add eslint config and npm script

### PR DESCRIPTION
**Pull request by glennhartmann**
Uses Google style for js and ts files. Has to be run manually at the moment, but eventually we can enforce this for PRs.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 13,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+    }
```
Comment (beverloo) [inline .eslintrc.json:18]: Do we want to open issues for discussing to add new rules? It's kind of fun to create our own coding guides :)

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "build": "webpack --mode=production",
     "serve": "webpack serve --open --mode=development",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint './**/*.js' './**/*.ts'"
```
Comment (beverloo) [inline package.json:9]: nit: should we try to keep this alphabetized? optional

**Review by beverloo — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (glennhartmann) [reply package.json]: done

File: .eslintrc.json (hunk 1 above)
Comment (glennhartmann) [reply .eslintrc.json:18]: sure, I'll open one in a minute

**Pull request merged**